clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferGvcfV1QueryOutput.scala

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.clio.transfer.model
+
+import java.time.OffsetDateTime
+
+import org.broadinstitute.clio.util.model.{DocumentStatus, Location}
+
+case class TransferGvcfV1QueryOutput(
+  analysisDate: Option[OffsetDateTime] = None,
+  contamination: Option[Float] = None,
+  documentStatus: Option[DocumentStatus] = None,
+  gvcfMd5: Option[String] = None,
+  gvcfPath: Option[String] = None,
+  gvcfSize: Option[Long] = None,
+  gvcfMetricsPath: Option[String] = None,
+  location: Location,
+  pipelineVersion: Option[String] = None,
+  project: String,
+  sampleAlias: String,
+  version: Int,
```

Comment:
Can you move the non-optional parameters to the top? It's the pattern we use in the wgs-ubam models.